File: examples/tokenbench/pipeline.py
users = [{"name": "ada", "age": 36}, {"name": "bob", "age": 17}, {"name": "cyd", "age": 24}]
adults = sorted((u for u in users if u["age"] >= 18), key=lambda u: u["name"])
print(", ".join(f"{u['name']}:{u['age']}" for u in adults))
print(sum(u["age"] for u in users))

File: examples/tokenbench/pipeline.ts
const users = [{ name: "ada", age: 36 }, { name: "bob", age: 17 }, { name: "cyd", age: 24 }]
const adults = users.filter(u => u.age >= 18).sort((a, b) => a.name.localeCompare(b.name))
console.log(adults.map(u => `${u.name}:${u.age}`).join(", "))
console.log(users.reduce((s, u) => s + u.age, 0))
